### PR TITLE
Task/TUI-3 - Add UIPatterns and associated components

### PR DESCRIPTION
**Pull request by Strmiska**
## Overview: ##
Updated UI Photo. There are still some things that aren't quite rendering as expected, but I'm working through them.

## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## UI Photos:
image

### CODE REVIEW

File: client/jest.config.js [hunk 1]
```diff
@@ -69,8 +69,8 @@ module.exports = {
   // A map from regular expressions to module names that allow to stub out resources with a single module
   moduleNameMapper: {
     '.*\\.(css|scss|sass)$': '<rootDir>/__mocks__/styleMock.js',
-    '^tapis-redux(.*)$': '<rootDir>/src/tapis-redux$1',
-    '^utils(.*)$': '<rootDir>/src/utils$1',
+    '^_common(.*)$': '<rootDir>/src/components/_common$1',
```
Comment (jchuahtacc) [inline client/jest.config.js:72]: Have both tapis-redux and _common

File: client/src/components/App/App.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
+.workbench-wrapper {
```
Comment (jchuahtacc) [inline client/src/components/App/App.scss:1]: The workbench-wrapper and workbench-content classes setup the sidebar and the content to the right, but should probably be renamed

File: client/src/components/Sidebar/Sidebar.module.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
+$width: 215px;
```
Comment (jchuahtacc) [inline client/src/components/Sidebar/Sidebar.module.scss:1]: Need to make sure the css modules webpack middleware is in the webpack config.

File: client/src/components/App/App.scss (hunk 2 above)
Comment (Strmiska) [reply client/src/components/App/App.scss:1]: Going to rename this once I can track down the issue with the rest of the styling.

**Comment by Strmiska** [comment]
The linting and unit tests pass locally, but do not seem to pass in the repo. Still need to figure out a couple more formatting issues with the sidebar navigation.

**Review by jchuahtacc — APPROVED** [review]
Adding additional issue and assigning with "ui patterns" tag
- Sidebar dom needs to be refactored to allow styles to be applied properly
- Buttons do not appear to have correct styles.

**Pull request merged**